Traceback (most recent call last):
  File "C:\Users\Olive\Desktop\oliveboard\learnings\errors_exceptions.py", line 3, in <module>
    raise Exception('This is the error message.')
Exception: This is the error message.
